PhotoEnhancer/PhotoEnhancer/Filters/SaturationParameters.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+
+namespace PhotoEnhancer
+{
+    public class SaturationParameters : IParameters
+    {
+        [ParameterInfo(Name = "Коэффициент увеличения/уменьшения насыщенности",
+                    MinValue = 0,
+                    MaxValue = 10,
+                    DefailtValue = 1,
+                    Increment = 0.05)]
+
+        public double Coefficient { get; set; }
```

Comment:
Атрибуты не принято отделять пустой строкой от класса / члена класса.
```suggestion
        [ParameterInfo(Name = "Коэффициент увеличения/уменьшения насыщенности",
                    MinValue = 0,
                    MaxValue = 10,
                    DefailtValue = 1,
                    Increment = 0.05)]
        public double Coefficient { get; set; }
```